### PR TITLE
whoami: added environment id response

### PR DESCRIPTION
**Pull request by devkeydet**

### CODE REVIEW

**Comment by paulbreuler** [comment]
Given the need to output the environment ID in many files can it be broken out into a single responsibility functional call?  Like GetEnvironmentId or some such? I’m not near my PC today but just poking around this PR so maybe I’ll connect with you later. 

I’m a fan of this addition.

**Comment by devkeydet** [comment]
@paulbreuler - not sure I follow your feedback.  Outside of the dist folder, these are the only files that have changed:
image

File: dist/actions/backup-environment/index.js [hunk 1]
```diff
@@ -1726,6 +1727,14 @@ function whoAmI(parameters, runnerParameters) {
             logger.log("The Authentication Result: " + authenticateResult);
             const pacResult = yield pac("org", "who");
             logger.log("WhoAmI Action Result: " + pacResult);
+            const envIdLabel = "Environment ID:";
+            // HACK TODO: Need structured output from pac CLI to make parsing out of the resulting env id more robust
```
Comment (paulbreuler) [inline dist/actions/backup-environment/index.js:1731]: @devkeydet, I was talking about the files with these lines of code. Under files changed I saw 26 modififcations with the same code across many files but with a TODO comment about needing better output from PAC CLI to make parsing easier. The snippet appears in 16 locations from a quick regex search on GitHub.

Comment (paulbreuler) [reply dist/actions/backup-environment/index.js:1731]: Sorry for the confusion I was on my mobile device earlier and didn't see the code tag feature there. A bit hard to follow text without some link to the code snippet :'(

**Comment by paulbreuler** [comment]
Ignore, it's an artifact of reviewing the changes from GitHub mobile. It wasn't clear that that was the dist folder output. I had just seen HACK TODO spammed in 16 files. Looking on a proper desktop view... yeah... my bad. Looks good to me.

**Comment by devkeydet** [comment]
@paulbreuler - I agree that the string 'environment-id' is in enough places that it should not be hard coded.
@davidjenni - I've gone ahead and updated the code to clean this up.  I didn't reference OutputVariables.EnvIdVariableName in other actions that use 'environment-id' because that's not work needed to accomplish the whoami enhancement.  Let me know if you want me to update other actions for consistency.  Whether in this PR or another.

**Comment by devkeydet** [comment]
@davidjenni - It wasn't clear to me what the process is to update the dist folder but looking at the history it looks like I probably should not have updated the dist folder.  The reason I did was because I also added validation in pull-request.yml to ensure the id matches that from create-environment.  Let me know if/how I should clean this up.

**Review by davidjenni — APPROVED** [review]
